### PR TITLE
integrate right orthogonal file by renumbering

### PR DESCRIPTION
**Pull request by jonweinb**
See https://github.com/rzk-lang/sHoTT/issues/87

### CODE REVIEW

File: src/simplicial-hott/05-segal-types.rzk.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ This is a literate `rzk` file:
 - `hott/total-space.md` — We rely on
   `#!rzk is-equiv-projection-contractible-fibers` and
   `#!rzk total-space-projection` in the proof of Theorem 5.5.
-- `3-simplicial-type-theory.md` — We rely on definitions of simplicies and their
+- `2-simplicial-type-theory.md` — We rely on definitions of simplicies and their
```
Comment (fredrik-bakke) [inline src/simplicial-hott/05-segal-types.rzk.md:19]: Seems to me the full file name should be used here

```suggestion
- `02-simplicial-type-theory.rzk.md` — We rely on definitions of simplicies and their
```

File: src/simplicial-hott/05-segal-types.rzk.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ This is a literate `rzk` file:
...
   subshapes.
-- `4-extension-types.md` — We use the fubini theorem and extension
+- `3-extension-types.md` — We use the fubini theorem and extension
```
Comment (fredrik-bakke) [inline src/simplicial-hott/05-segal-types.rzk.md:21]:
```suggestion
- `03-extension-types.rzk.md` — We use the fubini theorem and extension
```

**Comment by fredrik-bakke** [comment]
I noticed the extension `.rzk` is omitted for other files referenced in that document as well. Perhaps this is intentional?

**Comment by jonweinb** [comment]
Maybe just a leftover, feel free to go ahead and fix @fredrik-bakke (whenever you get around to it)

**Review by fredrik-bakke — APPROVED** [review]

**Review by TashiWalde — APPROVED** [review]

**Comment by emilyriehl** [comment]
Thanks all. I think this is ready to merge. I just added the new file to the web table of contents (by updating the mkdocs file).

**Pull request merged**